src/ZoneServer/Skills/Skill.cs

```diff
@@ -214,6 +214,26 @@ public void StartCooldown(TimeSpan cooldownTime)
 			Send.ZC_OVERHEAT_CHANGED(character, this);
 		}
 
+		/// <summary>
+		/// Resets the skill's cooldowns
+		/// </summary>
+		/// <param name="cooldownTime"></param>
+		public void ResetCooldown()
```

Comment:
Instead of copying the `StartCooldown` method, you could instead call it with a zero TimeSpan argument.